### PR TITLE
build login and signup pages

### PR DESCRIPTION
**Pull request by ishitakeshawani**
preview link:  https://deploy-preview-5--stunnershub.netlify.app/

### CODE REVIEW

File: pages/Authentication/signup/signup.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
+.signup {
+  border: 1px soild black;
```
Comment (aniketsingh13) [inline pages/Authentication/signup/signup.css]: use hex code for consistency

**Review by aniketsingh13 — APPROVED** [review]
look good to me

File: pages/Authentication/signup/signup.css (hunk 1 above)
Comment (ishitakeshawani) [reply pages/Authentication/signup/signup.css]: done

**Review by Creator-singh — APPROVED** [review]
looks good to me great work

**Comment by ishitakeshawani** [comment]
> looks good to me great work

Thank you :)

**Pull request merged**